packages/api/server.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express')
+const app = express()
+const port = process.env.PORT || 3000
+
+// Console.log to show server up and running in terminal
+app.listen(port, () => console.log('Listening on port ' + port + '...'))
+
+// Practice GET request to test server
+app.get('/beautifulportland', (req,res) => {
+    res.send({express: "The force is strong with this team..."})
+})
```

Comment:
Github complains here about the file not ending in a newline, which is something I'd never seen before, but I found this: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline
Sounds like a standard we'd want to adopt, especially since it could have consequences for future build tools we might use.
```suggestion
})

```